### PR TITLE
Glooscap

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary
Content "Abenaki hunter Glooscap"

#### Purpose of change

More diverse NPC types and stories, give some social traits a use in game, make Cataclysm feel like New England

#### Describe the solution

New NPC with Abenaki stories and name. Can be befriended or attacked many different ways. Written to work as either gender.

EDIT: Reworked mapgen stuff to include some content from #55501 to avoid merge troubles. Added shopkeep service after befriending, added another dialogue tree to befriend not gated behind INT 11.

#### Describe alternatives you've considered

Make Glooscap a follower but the follower system is not great. Give Glooscap some kind of mission for the player. Add breadcrumb missions like other holdouts have

#### Testing

Everything spawns no errors

Screen Shot 2022-03-14 at 8 52 45 PM

Trading works now
Screen Shot 2022-03-14 at 8 54 30 PM

#### Additional context

Inspired by the Glooscap stories of the Abenaki people https://en.wikipedia.org/wiki/Glooscap

### CODE REVIEW

**Comment by LyleSY** [comment]
Waiting on #55501 to merge to avoid merge troubles

**Comment by haveric** [comment]
Are you pretty much cloning the entirely of the lodge? If so, I feel like it might make sense to do some nested chunking here if they are intended to be nearly identical and that way we avoid these diverging when someone makes a change to one.

**Comment by LyleSY** [comment]
No, just the first and second floors. I'm open to that if there's a way to do it cleanly and still leave the door open for a breadcrumbs mission mapgen spawn later on. All I'm doing is adding the NPC downstairs, turning the zombies to pets, and adding a corpse upstairs.

**Comment by haveric** [comment]
If the map layout isn't changing, I believe that can be done and would simplify changes to yours. I've made my final changes to my branch and rebased it, so barring any conflicts, that will hopefully get approved and merged in the near future and then this can be re-evaluated. There might be an easier way as well to do what I'm thinking, but I feel like it makes sense if you're just modifying spawns.

File: data/json/npcs/holdouts/NC_GLOOSCAP.json [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    "dynamic_line": {
+      "u_has_var": "u_met_glooscap",
+      "type": "general",
+      "context": "meeting",
+      "value": "yes",
+      "yes": [
+        "Good to see you again, how can I help you?",
+        "They say I turned a man into a cedar tree once.  That's a terrible thing to say.",
+        "You like maple syrup?  That was me.  Not the syrup, that was the trees, but I figured out how to make it much harder to concentrate it, used to be way too easy.  Pretty nice right?",
+        "Did I tell you the one about the beavers?  Used to be way too big.  Like giant beavers, it was a mess.  I fixed that.  Of course there seem to be a bunch of new giant things out here now.  I'll have to think about that.",
```
Comment (Faalagorn) [inline data/json/npcs/holdouts/NC_GLOOSCAP.json:25]: Should there be a comma between "Like" and "giant"?

File: data/json/npcs/holdouts/NC_GLOOSCAP.json [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+      "no": "It's decided then.  I will be Glooscap.  Who will you be?"
+    },
+    "responses": [
+      {
+        "text": "Glooscap?  That's a funny name.  Is it foreign?",
+        "condition": { "not": { "u_has_var": "u_met_glooscap", "type": "general", "context": "meeting", "value": "yes" } },
+        "topic": "TALK_GLOOSCAP_WHERE"
+      },
+      {
+        "text": "[INT 11]  Like the Abenaki myth?",
```
Comment (Faalagorn) [inline data/json/npcs/holdouts/NC_GLOOSCAP.json:37]: There should be one space here

File: data/json/npcs/holdouts/NC_GLOOSCAP.json (hunk 1 above)
Comment (LyleSY) [reply data/json/npcs/holdouts/NC_GLOOSCAP.json:25]: Not in this case, but that would be appropriate for people who use "like" as a placeholder while they are thinking. I use "um" instead personally.

**Pull request merged**